app/models/like.rb

```diff
@@ -1,4 +1,8 @@
 class Like < ApplicationRecord
   belongs_to :post
   belongs_to :user
+
+  def self.update_likes_counter(post_id)
+    Post.increment_counter(:likes_counter, post_id)
+  end
 end
```

Comment:
- [ ] Again here it would be nice to use one of the Rails Active Callbacks - after_save to call the #update_likes_counter method every time a new like has been saved to database,